app/models/iiif_record.rb

```diff
@@ -0,0 +1,51 @@
+class IIIFRecord
+  def initialize(solr_doc)
+    @solr_doc = solr_doc
+  end
+
+  def to_json
+    { label: label, description: description, metadata: metadata }
+  end
+
+  def authors
+    authors_json = (@solr_doc['author_roles_1display'] || []).first
+    return {} unless authors_json
+    JSON.parse(authors_json)
+  end
```

Comment:
We can probably just map the primary creator to dc:creator and the other names to dc:contributor.  So I'll switch to mapping `author_display` to creator and map the other names from `related_name_json_1display` to contributor.